tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -76,6 +77,36 @@ def pre_insert_agents():
         assert data[0] == 'ok', f'Unable to update agent {id}'
 
 
+@pytest.fixture(scope="function")
+def pre_set_sync_info():
+    """Asign the last_attempt value to last_completion in sync_info table to force the synced status"""
+
+    command = 'agent 000 sql UPDATE sync_info SET last_completion = 10, last_attempt = 10 ' \
+              'where component = "syscollector-packages"'
+    receiver_sockets[0].send(command, size=True)
+    response = receiver_sockets[0].receive(size=True).decode()
+    data = response.split(" ", 1)
```

Comment:
```suggestion
    data = response.split(' ', 1)
```
For `split`, the default separator is any whitespace. If it is possible, we should use:

```suggestion
    data = response.split(1)
```